soh/src/code/z_eff_blure.c

```diff
@@ -394,7 +569,7 @@ void EffectBlure_GetComputedValues(EffectBlure* this, s32 index, f32 ratio, Vec3
             break;
     }
 
-    //sp30 = sp30; // Optimized out but seems necessary to match stack usage
+    sp30 = sp30; // Optimized out but seems necessary to match stack usage
```

Comment:
why was this uncommented?

Reply:
That's a good question, actually. I think it got uncommented in my mad hysteria relating to figuring out the trail code and I...guess I forgot to recomment it out.